renderer/containers/Onboarding/Steps/ConnectionDetails.js

```diff
@@ -19,6 +20,7 @@ const mapStateToProps = state => ({
   connectionHost: state.onboarding.connectionHost,
   connectionCert: state.onboarding.connectionCert,
   lndConnect: state.onboarding.lndConnect,
+  name: state.onboarding.name,
```

Comment:
A nice follow up PR would be to switch to using selectors here rather than accessing the state directly like this.